curriculum/challenges/english/07-scientific-computing-with-python/learn-tree-traversal-by-building-a-binary-search-tree/65c644829cfb63acf3479d09.md

```diff
@@ -1,61 +1,77 @@
 ---
-id: 655a45c52fa3ea5ece3034c7
+id: 65c644829cfb63acf3479d09
 title: Step 27
```

Comment:
why there is code appearing magically from nowhere here? it doesn't look correct... we should give it in the instructions and let the camper put it in the editor

Reply:
i have added another step for creating a list.